### DISCUSSION TITLE
Transfer parts from stock to another without merging

### DISCUSSION THREAD

**Discussion by MrShippeR** [inventree/InvenTree#6825] [Q&A]
Hello comunity,
I would like ask you how use system right. I have 2 stock locations:

- New items on transport to Central stock
- Central stock

### Idea
My way of using Inventree is: I Add items at PC web interface to "items on transport" and someone transport it with car to "Central stock". 
When parts arrive, I want someone get Android app and check in delivered parts (count, undamaged...). To assign them to shelf where the part belogs, where is QR code for item and some parts are already there. Operator then scan QR on shelf and it should increase count of existing item. I should see that increase in Item history.

### Reality
When someone change location of item from "items on transport" to "Central stock" there are 2 separate items and I must Merge them to one. And when I use Merge button, Item history is out/non complete/starting from blank.

### Question
Is there method how do this without merging? Idealy by scanning QRs?

**Comment by SchrodingersGat** [comment]
@MrShippeR to summarize:

You want a way to automatically merge a stock item with an existing one, when transferring from one location to another. If there is an existing stock item in that location, merge the transferred item with that one

**Comment by MrShippeR** [comment]
@elmo2k3 Yeah, right.

**Reply by SchrodingersGat** [reply]
Gotcha. This is not possible at the moment but it would make sense to add this as an option. I'll raise a new issue for this concept.
